public/app/plugins/datasource/cloudwatch/components/QueryEditor.tsx

```diff
@@ -161,7 +161,11 @@ export class QueryEditor extends PureComponent<Props, State> {
                   datasource.getDimensionKeys(query.namespace, query.region).then(this.appendTemplateVariables)
                 }
                 loadValues={newKey => {
-                  const { [newKey]: value, ...newDimensions } = query.dimensions;
+                  const { [newKey]: value, ...dim } = query.dimensions;
+                  const newDimensions = Object.entries(dim).reduce(
+                    (result, [key, value]) => (value === '*' ? result : { ...result, [key]: value }),
+                    {}
+                  );
```

Comment:
this part is a bit cryptic, maybe move loadValues into a seperate function, and add a comment about filtering out wildcard dimensions?

Reply:
👍